x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/analysis/VerifierTests.java

```diff
@@ -1087,6 +1088,13 @@ public void testMatchFilter() throws Exception {
         );
     }
 
+    public void testMatchFunctionNotAllowedAfterCommands() throws Exception {
+        assertEquals(
```

Comment:
do we want to add a `assumeTrue("skipping because MATCH is not enabled", EsqlCapabilities.Cap.MATCH_FUNCTION.isEnabled());` here?

Reply:
Thanks for the catch! Done in 141dddab6a29967837c04d08a403f389f99c4e28